upgrade.yml

```diff
@@ -575,5 +636,13 @@
   roles:
     - role: horizon
   environment: "{{ env_vars|default({}) }}"
+# Use haproxy port for post upgrade
+- name: set mysql connection string 3307
+  hosts: all:!vyatta-*
+  tasks:
+    - name: set mysql endpoint port 3307
+      set_fact:
+        endpoints:
+          db: "{{ undercloud_floating_ip }}:3307"
 
 - include: site.yml
```

Comment:
Let's work this under new PR. 
- if we remove it than it will impact upgrade workflow where operator has to run 2 ursula runs
  As @scoop206 mentioned, it does help to easily restart the play based on where it failed.
- I like approach of putting in debug message to call out is now into site.yml play and still leave it in upgrade.yml